source/lib/rules/files-property.ts

```diff
@@ -36,3 +39,29 @@ export default (context: Context): Diagnostic[] => {
 		}
 	];
 };
+
+function processGitIgnoreStylePatterns(patterns: string[]): string[] {
+	const processedPatterns = patterns.reduce((includedPatterns, pattern) => {
```

Comment:
I would prefer to not use `.reduce`. It makes code hard to read. Usually, you can solve it with a `.filter`, `.map`, or for-of instead.

Reply:
done